src/backend/utils/fileStorageHelper.ts

```diff
@@ -0,0 +1,83 @@
+import fs from "fs"
+import fsPromises from "fs/promises"
+import path from "path"
+import { tryCatch } from "./tryCatch"
+
+export const makeFileStorageHelper = (rootStorage: string = "./storage") => {
```

Comment:
ehh my design here is pretty weird i'll admit.  These may as well be separate functions.  it is useful though that it's a plaintext file storage writer to a specific directory that can initialize itself.  so I've kept it around.